pkgs/development/python-modules/pdfminer-six/default.nix

```diff
@@ -1,18 +1,20 @@
 { lib
 , buildPythonPackage
 , fetchFromGitHub
+, importlib-metadata
 , isPy3k
 , cryptography
 , charset-normalizer
 , pythonOlder
 , typing-extensions
 , pytestCheckHook
+, substituteAll
 , ocrmypdf
 }:
 
 buildPythonPackage rec {
   pname = "pdfminer-six";
-  version = "20221105";
+  version = "20231228";
   format = "setuptools";
```

Comment:
Could you please replace this with `pyproject = true` to remove a legacy build?
Besides this, LGTM.